tests/TestCase/Database/SchemaCacheTest.php

```diff
@@ -51,6 +51,7 @@ public function setUp()
     {
         parent::setUp();
 
+        Cache::enable();
```

Comment:
What is this needed for?

Reply:
It is wired, but the test doesn't pass on my local environment as Cache is disabled if I run all tests. Probably some other test would disable Cache and not restore the settings. But I should fix the root cause. Let me try finding it.

Reply:
ok